src/app/frontend/deploy/deployfromsettings.scss

```diff
@@ -14,6 +14,15 @@
 
 @import '../variables';
 
+* {
+  font-size: 14px;
```

Comment:
It changes font size not only here, but on other pages too. It has to be changed to specific class.